ui/src/routes/devices.$id.tsx

```diff
@@ -211,7 +212,7 @@ export default function KvmIdRoute() {
           clearInterval(checkInterval);
           setLoadingMessage("Connection established");
         } else if (attempts >= 10) {
-          console.log(
+          console.warn(
```

Comment:
Reduced some logging noise and emit warnings/errors when appropriate so the console log filter is useful